compiler/literate/html.ml

```diff
@@ -200,6 +200,7 @@ let rec law_structure_to_html
          (law_structure_to_html language print_only_law complete_headings))
       children fmt_details_close ()
   | A.LawInclude _ -> ()
+  | A.ModuleDef _ | A.ModuleUse _ -> () (* TODO: show somehow ?*)
```

Comment:
Yes we should show it. I would be in favor of collecting all module def and use and show a summary of them at the very beginning, after the list of files weaved for the current module.